src/gen/conditions.rs

```diff
@@ -3311,3 +3371,119 @@ fn test_multiple_my_birth_assertions(
 
     test(spend);
 }
+
+#[test]
+fn test_assert_ephemeral() {
+    // ASSERT_EPHEMERAL
+    // the coin11 value is the coinID computed from (H1, H1, 123). The first
+    // coin we spend in this case
```

Comment:
```suggestion
    // the coin11 value is the coinID computed from (H1, H1, 123). 
    // coin11 is the first coin we spend in this case.
```

What you had was correct - I find this easier to read because the sentence was running into the "51 is ..." comment below.